app/src/main/java/com/example/myanimereport/fragments/HomeFragment.java

```diff
@@ -70,7 +78,7 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
     public void queryEntries(int skip) {
         ParseQuery<Entry> query = ParseQuery.getQuery(Entry.class); // Specify type of data
         query.setSkip(skip); // Skip the first skip items
-        query.setLimit(20); // Limit query to 20 items
+        query.setLimit(10); // Limit query to 20 items
```

Comment:
nit: you can update the comment here to reflect the updated query limit, or vice versa